spec/link/group-subgroup.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  type: 'link/group-subgroup',
+  tangle: 'link',
+  staticProps: {
+    parent: { $ref: '#/definitions/cloakedMessageId', required: true },
+    child: { $ref: '#/definitions/cloakedMessageId', required: true }
+  },
+  props: {
+  }
```

Comment:
should i add a name prop on here too? like `feed-group` links

Reply:
no not yet. I think name was used mainly so people can tag a group as `__personal__` to make it easy to find a personal group